### ISSUE TITLE
Backup whole cluster - alternative

### ISSUE THREAD

**Issue by rsantosdev** [rethinkdb/rethinkdb#5954]
We have a 3 machine cluster running on docker cloud (AWS host). We spin a container to make backups every hour of our database. Everything is working fine.

Except that we have a small database and during the bkp time the reads go up 15k and makes major of system queries to run slowly.

The bkp log:
```
1476078 rows exported from 25 tables, with 70 secondary indexes
Done (213 seconds)
```

The data of exporting some key tables
```
{
  "new_val": {
    "duration_sec": 193.228456,
    "id": [
      "query",
      "17b3296e-e380-412a-b970-4e23416de7d4"
    ],
    "info": {
      "client_address": "::ffff:10.7.0.47",
      "client_port": 53419,
      "query": "r.db().table().orderBy({\"index\": \"id\"})"
    },
    "servers": [
      "rethinkdb_1"
    ],
    "type": "query"
  },
  "old_val": {
    "duration_sec": 192.226475,
    "id": [
      "query",
      "17b3296e-e380-412a-b970-4e23416de7d4"
    ],
    "info": {
      "client_address": "::ffff:10.7.0.47",
      "client_port": 53419,
      "query": "r.db().table().orderBy({\"index\": \"id\"})"
    },
    "servers": [
      "rethinkdb_1"
    ],
    "type": "query"
  }
}
```

My question is since I have a single database in my cluster and I want always bkp the full database, would be better bkp the filesystem in this case ?! Is this possible ?! How can I recover the database in that scenario ?!

**Comment by larkost** [comment]
You could setup a non-voting replica that would mirror all of the data, and then take that down when you wanted to back it up. It would only add a small amount of traffic to the cluster during writes, and could be taken offline without consequences. Once the server was down you could back up the data folder.

However, this is not going to be easy to use when you have a problem, as we are setup more for recovering from backups made with `dump`. It would be possible, but you are going to have to play with server assignments, and possibly force the new configuration, so this is not going to be a route I would encourage if something else is possible.

And to that end I have just filed two new bugs: #5959 and #5960. Either one of them would be the basis of a good solution for you. I will try and complete both of them next week (neither should be much work). They will probably wait a little bit after that to get released, as my next task is to #2598 (testing for `dump` and the like).

@rsantosdev would you be willing to help me test the new drivers when they are ready?

And since there is no bug in this report, or new proposals I am going to close it as not-a-bug. Feel free to keep discussing here.

**Comment by rsantosdev** [comment]
@larkost sure man. I would be more than happy to test the solution with you. just ping me here or on slack. My username is the same over there
